go-controller/pkg/ovn/policy.go

```diff
@@ -901,60 +898,36 @@ func hasAnyLabelSelector(peers []knet.NetworkPolicyPeer) bool {
 	return false
 }
 
-// addNetworkPolicy creates and applies OVN ACLs to pod logical switch
-// ports from Kubernetes NetworkPolicy objects using OVN Port Groups
-func (oc *Controller) addNetworkPolicy(policy *knet.NetworkPolicy) {
-	klog.Infof("Adding network policy %s in namespace %s", policy.Name,
-		policy.Namespace)
-
-	nsInfo, err := oc.waitForNamespaceLocked(policy.Namespace)
+// only call with nsInfo lock
+func (oc *Controller) ensureDefaultDenyPortGroup(nsInfo *namespaceInfo, ns string, policyName string) error {
```

Comment:
the word "ensure" is used around the code to say "if the object doesn't exist make it". This doesn't right? So it's just trying to make the default deny PG without looking to see if it exists first which is a bit confusing

Reply:
good point, i'll change the name